app/src/main/java/com/example/guineapics/RecyclerViewAdapter.java

```diff
@@ -0,0 +1,43 @@
+package com.example.guineapics;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.guineapics.dao.GuineaPigsDAO;
+
+public class RecyclerViewAdapter extends RecyclerView.Adapter<TileViewHolder> {
+    private LayoutInflater inflater;
```

Comment:
you dont need to have this in a class variable, LayoutInflater has this static method that you can use every time

Reply:
What do you mean? That was in Google Codelab and I can't see any other option